mmv1/products/cloudfunctions2/api.yaml

```
@@ -0,0 +1,302 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CloudFunctions2
```

```suggestion
name: Cloudfunctions2
```

Capital letters are load bearing in this string, so the resource name is currently `google_cloud_functions2_cloud_function`. This change makes it `google_cloudfunctions2_cloud_function` as you've specified in the example.

---

Updated to read google_cloudfunctions2_function